quantum/process_keycode/process_midi.h

```diff
@@ -3,205 +3,38 @@
 
 #include "quantum.h"
 
-bool process_midi(uint16_t keycode, keyrecord_t *record);
+#ifdef MIDI_ENABLE
+
+#ifdef MIDI_BASIC
+void process_midi_basic_noteon(uint8_t note);
+void process_midi_basic_noteoff(uint8_t note);
+void process_midi_basic_stop_all_notes(void);
```

Comment:
this should probably be named `process_midi_basic_all_notes_off` to align better with names used in `process_music.c`